pkgs/development/compilers/cudatoolkit/flags.nix

```diff
@@ -60,6 +60,12 @@ let
   # supportedCapabilities :: List Capability
   supportedCapabilities = lists.map (gpu: gpu.computeCapability) supportedGpus;
 
+  # unsupportedCapabilities :: List Capability
+  unsupportedCapabilities =
+    lists.filter
+    (computeCapability: !lists.elem computeCapability supportedCapabilities)
+    (config.cudaCapabilities or defaultCapabilities);
+
```

Comment:
Since this depends on `defaultCapabilities`, can we move it to be after `defaultCapabailties`'s definition? I know that it runs fine without it, but IMHO it's more intuitive to keep definitions in topo-sorted order.

Reply:
Will do!